app/views/definition.scala.txt

```diff
@@ -5,7 +5,7 @@
     {
       "key": "@apiScopeKey",
       "name": "API Notification Pull",
-      "description": "Pull notifications"
+      "description": "Pull notificationzs"
```

Comment:
Is the spelling correct?
You added a `z`

Reply:
Oops, I think a cat must have walked across my keyboard!